src/k8s-extension/azext_k8s_extension/_help.py

```diff
@@ -83,41 +83,79 @@
     short-summary: Commands to discover Kubernetes Extension Types.
 """
 
-helps[f'{consts.EXTENSION_NAME} extension-types list'] = f"""
+helps[f'{consts.EXTENSION_NAME} extension-types get-by-cluster'] = f"""
     type: command
-    short-summary: List Kubernetes Extension Types.
+    short-summary: Get properties for a Cluster Extension Type for an existing cluster.
```

Comment:
>for an existing cluster [](http://example.com/codeflow?start=63&length=23)

Expand this some more to clarify how the cluster information is used in filtering - I believe it is by the clusterType AND the region?  Are there other properties of the cluster used in filtering the results?

Reply:
Done